### PR TITLE
release: release version 0.8.0

### PR DESCRIPTION
**Pull request by martinvonz**
# Checklist

If applicable:
- [x] I have updated `CHANGELOG.md`
- [x] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Comment by martinvonz** [comment]
This should be ready to merge, but we can wait for a week or so if we want to see if we discover any problems with new crates names that we want to fix before we cut a release.

File: README.md
Comment (omasanori) [inline README.md:3]:
```suggestion
![](https://img.shields.io/github/license/martinvonz/jj) ![](https://img.shields.io/github/v/release/martinvonz/jj) ![](https://img.shields.io/github/release-date/martinvonz/jj) ![](https://img.shields.io/crates/v/jj-cli)
```

**Review by omasanori — CHANGES_REQUESTED** [review]
I have a nitpick. Thank you for your effort and I can't wait a new version!

File: README.md
Comment (martinvonz) [reply README.md]: Good catch! Fixed.

**Review by omasanori — APPROVED** [review]
Thanks for your response!

**Comment by martinvonz** [comment]
> This should be ready to merge, but we can wait for a week or so if we want to see if we discover any problems with new crates names that we want to fix before we cut a release.

It's now been a week, so I guess I'll go ahead and merge this and start the release.

**Comment by martinvonz** [comment]
> It's now been a week, so I guess I'll go ahead and merge this and start the release.

Once it's been approved by someone with write access, that is :)

**Review by PhilipMetzger — APPROVED** [review]
:-), atleast a contribution for `0.8`

**Pull request merged**
